bin/run-tc-ru_transfer.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
```

Comment:
That looks good, but we lack TC integration here. Should not be too hard, you could:
 - write a `taskcluster/tc-transfer-tests.sh` by looking at `taskcluster/tc-train-tests.sh`
 - add some YAML description inspired from `taskcluster/test-training_16k-linux-amd64-py36m-opt.yml`